lib/src/visibility_text.dart

```diff
@@ -0,0 +1,24 @@
+import "package:flutter/material.dart";
+import 'card_constants.dart';
+
+class VisibilityText extends StatelessWidget {
+  const VisibilityText(this.cardAttribute, this.visibleCondition, {Key? key})
```

Comment:
Why this attributes are outside {Key? key}?

Reply:
Missed trailing comma

Reply:
Done.